config/app/setup/handlers/setup.json

```
@@ -63,8 +63,8 @@
         "templateFolder": "@css:templates/root/empty",
         "factory": { "@type": "ExtensionBasedMapperFactory" },
         "templateEngine": {
-          "@type": "HandlebarsTemplateEngine",
-          "baseUrl": { "@id": "urn:solid-server:default:variable:baseUrl" }
+          "@type": "GenericTemplateEngine",
+          "handler": { "@id": "urn:solid-server:default:GenericTemplateEngine" }
```

```suggestion
           "@id": "urn:solid-server:default:GenericTemplateEngine" 
```
This does exactly the same right? So in all cases where there is no defined `template` parameter there is no need to instantiate a `GenericTemplateEngine`.